content/en/docs/components/training/admin-guides/installation.md

```diff
@@ -0,0 +1,58 @@
++++
+title = "Installation"
+description = "How to install the Kubeflow Training control plane"
+weight = 10
++++
+
+This guide describes how to install the Kubeflow Training control plane on a Kubernetes cluster.
+
+You can skip these steps if [the Kubeflow platform](https://www.kubeflow.org/docs/started/installing-kubeflow/)
+is already deployed using manifests or package distributions, as it includes Kubeflow Training by default.
+
+## Prerequisites
+
+These are the minimal requirements to install the Kubeflow Training control plane:
+
+- Kubernetes >= 1.28
+- `kubectl` >= 1.28
+
+{{% alert title="Tip" color="primary" %}}
+If you don't have Kubernetes cluster, you can quickly create one locally using [Kind](https://kind.sigs.k8s.io/docs/user/quick-start#installing-with-a-package-manager):
+
+```bash
+brew install kind
```

Comment:
You assume users have Mac?

Reply:
It is a good point, I just assume that many Data Scientists who use Mac might be using `brew` already.
Any other thoughts here @vsoch @kubeflow/wg-training-leads @astefanutti ?

Reply:
I agree we can assume Mac users might be using Brew, but at that stage I think it's about documenting how to create a Kind cluster, not installing Kind, which is provided by the link. So here keeping `kind create cluster` may be just enough and not exclude non-Mac users :)